plugins/autocomplete/plugin.js

```diff
@@ -172,6 +167,13 @@
 		 */
 		this.editor = editor;
 
+		/**
+		 * See {@link CKEDITOR.plugins.autocomplete.configDefinition#throttle}.
+		 *
+		 * @property {Number} [throttle=0]
```

Comment:
I'd avaoid duplicating the default value here. Keep it only within `configDefinition#throttle` instead.

Reply:
I mean code:

> `config.throttle || 0;`

Is a good defensive way of coding, and that's fine. My concern is only related to API docs.

> `@property {Number} [throttle=0]`